eve/workers/openstack-terraform/terraform/nodes.tf

```diff
@@ -157,23 +157,23 @@ resource "openstack_compute_instance_v2" "bootstrap" {
       "sudo chmod +x scripts/network.sh",
       "sudo scripts/network.sh eth1",
       "sudo scripts/network.sh eth2",
-      "sudo systemctl restart network"
+      "sudo systemctl restart network || sudo systemctl restart NetworkManager"
     ]
   }
 
   # Register RHSM if OS = rhel
   provisioner "remote-exec" {
     inline = [
-      "if [ '${var.os}' = 'rhel-7' ]; then sudo chmod +x scripts/rhsm-register.sh; fi",
-      "if [ '${var.os}' = 'rhel-7' ]; then sudo scripts/rhsm-register.sh '${var.rhsm_username}' '${var.rhsm_password}'; fi"
+      "case '${var.os}' in rhel-*) sudo chmod +x scripts/rhsm-register.sh;; esac",
+      "case '${var.os}' in rhel-*) sudo scripts/rhsm-register.sh '${var.rhsm_username}' '${var.rhsm_password}';; esac"
```

Comment:
Nothing, but terraform is using good old sh, so no bashism is available hence the use of `case` to do some glob comparison.
I could have used a `[ -n ${var##pattern} ]` construction also, but I find this one easier to read and it would need an extra step to store `${var.os}` is a shell var first..